test/e2e/delete_alert_rule_test.go

```
@@ -77,61 +77,73 @@ func TestDeleteAlertRule(t *testing.T) {
+		}
 
-	resp, err := f.HTTPClient().Do(req)
-	if err != nil {
-		t.Fatalf("Failed to make delete request: %v", err)
-	}
-	defer resp.Body.Close()
+		resp, err := f.HTTPClient().Do(req)
+		if err != nil {
+			return fmt.Errorf("failed to make delete request: %w", err)
+		}
+		defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)